### PR TITLE
chore: [IA-840] Changes to Help flow

### PR DESCRIPTION
**Pull request by AliceAzzolini**
THIS PR DEPENDS ON #3992 
copy changes intended to make the CTAs clearer to the users.

### CODE REVIEW

**Review by Undermaken — APPROVED** [review]

**Review by thisisjp — APPROVED** [review]

**Comment by thisisjp** [comment]
@Undermaken the first task of this story is completed, we're now missing the "dev" part. Please wait for the final approval\merge 💙

**Review by debiff — APPROVED** [review]

**Pull request merged**
